db/seeds.rb

```diff
@@ -1,10 +1,16 @@
 comedies = Category.create name: "Comedies"
 dramas = Category.create name: "Dramas"
 
-7.times { Video.create title: "Monk", description: "Adrian Monk is a brilliant San Francisco detective, whose obsessive compulsive disorder just happens to get in the way.", small_cover_url: "monk", large_cover_url: "monk_large", category: dramas }
+family_guy = Video.create title: "Family Guy", description: "In a wacky Rhode Island town, a dysfunctional family strive to cope with everyday life as they are thrown from one crazy scenario to another.", small_cover_url: "family_guy", large_cover_url: "family_guy_large", category: comedies
 
-Video.create title: "Family Guy", description: "In a wacky Rhode Island town, a dysfunctional family strive to cope with everyday life as they are thrown from one crazy scenario to another.", small_cover_url: "family_guy", large_cover_url: "family_guy_large", category: comedies
+7.times { Video.create title: "Monk", description: "Adrian Monk is a brilliant San Francisco detective, whose obsessive compulsive disorder just happens to get in the way.", small_cover_url: "monk", large_cover_url: "monk_large", category: dramas }
```

Comment:
Better way to format this would be:

```ruby
7.times do
  Video.create title: "...",
               description: "...",
               another_attribute: "...",
               whatever: "..."
end

```